tests/unit/wc.handlers.test.ts

```diff
@@ -277,5 +291,79 @@ Challenge: 4113fc3ab2cc60f5d595b2e55349f1eec56fd0c70d4287081fe7156848263626`
         "0x491e245db3914b85807f3807f2125b9ed9722d0e9f3fa0fe325b31893fa5e693387178ae4a51f304556c1b2e9dd24f1120d073f93017af006ad801a639214ea61b"
       );
     });
+
+    it("recovering eth_signTypedData", async () => {
```

Comment:
Could we add a TODO (and issue) about finding an actual payload for this (also `eth_sign`) - like figure out which Ethereum apps are actually making these kinds of requests.